doc/website/stylesheets/extra.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+    font-family: "TStar Pro Regular";
+    src: url("../fonts/TSTARPRO-Regular.otf") format("opentype");
+}
+
+@font-face {
+  font-family: "TStar Pro Bold";
+  src: url("../fonts/TSTARPRO-Bold.otf") format("opentype");
+}
+
+@font-face {
+    font-family: "TStar Pro Headline";
+    src: url("../fonts/TSTARPRO-Headline.otf") format("opentype");
+}
+
+@font-face {
+  font-family: "TStar Pro Light";
+  src: url("../fonts/TSTARPRO-Light.otf") format("opentype");
+}
+
+@font-face {
+    font-family: "TStar Pro Medium";
+    src: url("../fonts/TSTARPRO-Medium.otf") format("opentype");
+}
+
+@font-face {
+  font-family: "Slate Pro";
+  src: url("../fonts/Monotype-SlatePro.otf") format("opentype");
+}
+
+@font-face {
+    font-family: "Slate Pro Medium";
+    src: url("../fonts/Monotype-SlatePro-Medium.otf") format("opentype");
+}
+
+:root {
+    --md-primary-fg-color:        #2f8db3;
+    --md-primary-fg-color--light: #FFFFFF;
+    --md-primary-fg-color--dark:  #8dc6d9;
+    --md-accent-fg-color: #ff4d00;
+}
+
+body {
+    font-family: "TStar Pro Regular", sans-serif;
+}
+
+h1 {
+    font-family: "Slate Pro", sans-serif;
+    text-transform: uppercase;
+}
+
+h2 {
+    font-family: "TStar Pro Bold", sans-serif;
+}
+
+strong {
+    font-family: "TStar Pro Bold", sans-serif;
+}
+
```

Comment:
Could you add this and check if the main pane is not limited to approximately 800px anymore?
```
.md-grid {
    margin-right: auto;
    margin-left: auto;
}
```